src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowBuilder.java

```diff
@@ -447,5 +452,26 @@ String getReplacementVendorTaxNumber(String payeeId, T summary) {
         
         return stats;
     }
+    
+    protected int getMaxSearchSize() {
+    	if (maxSearchResultSize == null) {
+	    	String searchLimit = getParameterService().getParameterValueAsString(KewApiConstants.KEW_NAMESPACE, 
+	        		KRADConstants.DetailTypes.DOCUMENT_SEARCH_DETAIL_TYPE, KewApiConstants.DOC_SEARCH_RESULT_CAP);
+	    	try {
+	    		LOG.debug("Found a value for KewApiConstants.DOC_SEARCH_RESULT_CAP, and it is '" + searchLimit + "'");
+	    		maxSearchResultSize = Integer.parseInt(searchLimit);
+	    	} catch (Exception e) {
+	    		LOG.error("Unable to convert '" + searchLimit + 
+	    				"' to an integer.  Returning value of KewApiConstants.DOCUMENT_LOOKUP_DEFAULT_RESULT_CAP which is " + 
+	    				KewApiConstants.DOCUMENT_LOOKUP_DEFAULT_RESULT_CAP);
+	    		maxSearchResultSize = KewApiConstants.DOCUMENT_LOOKUP_DEFAULT_RESULT_CAP;

```

Comment:
Another implicit int-to-Integer conversion. It should work fine as-is, but I would suggest updating things for consistency due to the explicit Integer-to-int conversion on the return statement.